code/modules/medical/pathology/pathogen_machines.dm

```diff
@@ -757,6 +756,7 @@
 			var/dir = text2num(href_list["dir"])
 			if(mut_type && dir && (src.manip.machine_state == PATHOGEN_MANIPULATOR_STATE_MANIPULATE) && !(manipulating))
 				manipulating = true
+				var/manip_cooldown = src.manip.loaded.reference.maliciousness<65? 1 DECI SECONDS : 20 DECI SECONDS
```

Comment:
Wouldn't this let you still spam it at say, 49 maliciousness to gain random (weaker) symptoms, but 4 times faster then before?

Reply:
That is a very good point, I will adjust the formula to curtail this.

Reply:
The delay is now 1 with maliciousness below 15 (where you do not gain symptoms), 10 between 15 and 65 (where the odds are worse) and 20 above 65.